samples/matter/bridge/src/bridged_device_types/temperature_sensor.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+#include "bridged_device.h"
+
+using namespace chip;
```

Comment:
Overall, `using namespace` should be avoided in headers (because one needs to include both this header and another that have functions with the same name, there's no way to resolve that).

Reply:
Fixed